### PR TITLE
use 24h values; convert stroops to lumens; small bug cleanup

### PR DESCRIPTION
**Pull request by piyalbasu**

### CODE REVIEW

File: src/components/SortableTable/index.tsx [hunk 1]
```diff
@@ -120,30 +121,20 @@ export const SortableTable = <DataItem extends Record<string, any>>({
...
-                      to={item.href}
-                    >
-                      <RowContents />
-                    </Link>
-                  ) : (
-                    <RowContents />
-                  )}
                 </tr>
-              );
-            })}
```
Comment (piyalbasu) [inline src/components/SortableTable/index.tsx:146]: I wanted use anchor tags for screen readers, but it was throwing valid DOMnesting error. Bummer

File: src/helpers/formatPool.ts [hunk 2]
```diff
@@ -19,17 +15,30 @@ type Asset = {
...
     assetCode: getAssetCode(i.asset, "-"),
     ...i,
   }));
 
+const convertObjValToLumens = (item: LiquidityPoolAssetInterval) => {
+  const formattedItem = {} as {
+    [key: string]: any;
+  } & LiquidityPoolAssetInterval;
+  Object.entries(item).forEach(([k, v]) => {
+    formattedItem[k] = typeof v === "number" ? fromStroopsToLumen(v) : v;
```
Comment (piyalbasu) [inline src/helpers/formatPool.ts:29]: this is a little generic to handle a situation where stellar.expert adds more denominations (12h, 10d, 14d, etc)

File: src/components/DetailsChart/index.tsx [hunk 3]
```diff
@@ -30,7 +29,7 @@ export const DetailsChart = ({
     const formattedData = poolHistory.data.map((entry) => ({
       x: new Date(entry.ts * 1000).getDate().toString(),
       // TODO: total_value_locked not in history data
```
Comment (quietbits) [inline src/components/DetailsChart/index.tsx:31]: Do we still need this TODO?

File: src/types/types.d.ts [hunk 4]
```diff
@@ -98,7 +96,7 @@ export interface LiquidityPoolTransaction {
 export type LiquidityPoolHistory = {
   ts: number;
   /* eslint-disable camelcase */
```
Comment (quietbits) [inline src/types/types.d.ts:98]: We might not need these `eslint` comments anymore.

**Review by quietbits — APPROVED** [review]

**Pull request merged**